onnxruntime/test/providers/cpu/math/matmul_integer_test.cc

```diff
@@ -32,6 +33,203 @@ TEST(MatmulIntegerOpTest, MatMulInteger) {
   test.AddOutput<int32_t>("T3", {1, 1}, {-1});
   test.Run();
 }
+
+// clang-format off
+TEST(MatmulIntegerOpTest, MatMulInteger_int8_t) {
+  OpTester test("MatMulInteger", 10);
+  test.AddInput<int8_t>("T1",
+                        {2, 4},
+                        {-3,  7, 5, -6,
+                          4, -5, 8,  7});
+  test.AddInput<int8_t>("T2",
+                        {4, 4},
+                        { 5,  -3, 7,  8,
+                         -6, -8, -3,  6,
+                          7,  9,  9, -5,
+                          8,  7, -6,  7});
+  test.AddInput<int8_t>("a_zero_point", {}, {5});
+  test.AddInput<int8_t>("b_zero_point", {}, {5});
+  test.AddOutput<int32_t>("T3",
+                          {2, 4},
+                          { -55,  16, 89, -44,
+                            122, 154, 68, -39});
+
+  std::vector<std::unique_ptr<IExecutionProvider>> execution_providers;
+  execution_providers.push_back( DefaultCudaExecutionProvider() );
+  test.Run( OpTester::ExpectResult::kExpectSuccess, "", {}, nullptr, &execution_providers );
```

Comment:
Usually we run all tests with the CPU EP as well. Should it be added here as well to validate that output from the CPU implementation matches the CUDA implementation?

Reply:
Unsupported on CPU.